content/documentation/variables.json

```diff
@@ -2,11 +2,14 @@
   "tanzu-app-service-version": "2-11",
   "streaming-apps-version": "2021.0.1",
   "task-version": "2.4.5",
-  "skipper-version": "2.9.0-SNAPSHOT",
```

Comment:
The Microsite docs are only available per actual release (not milestones nor snapshot). These variables need to reflect that. I will add a section in the release-process doc that points to this file to make sure it is updated as part of the relesae.